src/encounters/LuanaEncounter.js

```diff
@@ -4,11 +4,49 @@ import PropTypes from 'prop-types';
 
 import Encounter from './Encounter';
 import { addTreat } from '../reducers/gameReducer';
+import { getActiveSprite } from '../reducers/spriteReducer';
+
+import { CANONICAL_SPRITE_COATS, SPRITE_COATS, SPRITE_ENCYCLOPEDIA } from '../gameData/spriteEncyclopedia';
+
+import { randomChoice } from '../helpers/utils';
+import { _n } from '../gameData/textHelpers/helpers';
+
+const goalSprite = () => {
```

Comment:
Can this re-use functionality that's used by the wilderness to generate a random sprite?